nixos/modules/services/monitoring/rustdesk-server.nix

```diff
@@ -0,0 +1,92 @@
+{ lib, pkgs, config, ... }:
+{
+  options.services.rustdesk-server = with lib; with types; {
+    enable = mkEnableOption "enable RustDesk";
```

Comment:
"enable RustDesk" -> "RustDesk, a remote access and remote control software, allowing maintenance of computers and other devices".